product_docs/docs/edbcloud/beta/getting_started/02_connect_cloud_account.mdx

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Connecting Your Cloud Account"
+title: "Connecting your cloud account"
```

Comment:
Would "cloud" here be a proper noun?

Reply:
I changed it to the actual product name EDB Cloud